src/App.js

```diff
@@ -224,13 +229,14 @@ const returnStartedRoute = (Component, props, rest) => {
   // The wallet is already loaded
   const routeRequiresWalletToBeLoaded = rest.loaded;
   if (LOCAL_STORE.getWalletId()) {
+    const isServerScreen = props.match.path === '/server';
     // Wallet is locked, go to locked screen
-    if (LOCAL_STORE.isLocked()) {
+    if (LOCAL_STORE.isLocked() && !isServerScreen) {
```

Comment:
Even a locked wallet can access the server screen if the server is unresponsive and the user wants to change the server.